docs/cloud-native-security/environment-variable-capture.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[environment-variable-capture]]
+
+== Environment variable capture
+
+.Requirements
+[sidebar]
+--
+* This feature requires {stack} version 8.6 or higher.
+* In {stack} version 8.6, this feature is only available for Linux.
+--
+
+You can configure an {agent} policy to capture up to five environment variables ("env vars"). Captured env vars are associated with process events and recorded in the `process.env_vars` field.
```

Comment:
```suggestion
You can configure an {agent} policy to capture up to five environment variables (`env vars`). Captured environment variables are associated with process events and recorded in the `process.env_vars` field.
```

Reply:
I'm fairly certain we style environment variable field using the code font style. Also suggested using the full term in the second sentence for clarity.